.storybook/main.ts

```diff
@@ -26,6 +26,12 @@ const config: StorybookConfig = {
               },
             ],
           },
+
+          {
+            test: /\.(ts|tsx)$/,
+            loader: 'esbuild-loader',
+            options: { tsconfig: './tsconfig.json', target: 'es2020' },
```

Comment:
Maybe this is why my import wasn't working? 🤔

Reply:
Yes that's it, I forgot with all of the changes